modules/textures/docs/api-reference/compressed-texture-loader.md

```diff
@@ -34,6 +34,6 @@ Returns an array of image data objects representing mip levels.
 
 ## Options
 
-| Option | Type | Default | Description |
-| ------ | ---- | ------- | ----------- |
-| N/A    |      |         |             |
+| Option                        | Type    | Default | Description                            |
+| ----------------------------- | ------- | ------- | -------------------------------------- |
+| `compressed-texture.useBasis` | Boolean | `false` | Use BasisLoader to decode KTX2 texture |
```

Comment:
Thanks for documenting it. I am not sure we want this to be a public option, but good for now. We should audit the API before 3.1 release.